tests/common/scripts/bookinfo.txt

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+
+# Copyright 2020 Istio Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+set -u
+set -o pipefail
+
+# Step 1: setup bookinfo & sleep pods
+kubectl label namespace default istio-injection=enabled --overwrite || true
+
+kubectl apply -f samples/bookinfo/platform/kube/bookinfo.yaml
+kubectl apply -f samples/bookinfo/networking/bookinfo-gateway.yaml
+kubectl apply -f samples/bookinfo/networking/destination-rule-all.yaml
+kubectl apply -f samples/sleep/sleep.yaml
+
+for deploy in "productpage-v1" "details-v1" "ratings-v1" "reviews-v1" "reviews-v2" "reviews-v3" "sleep"; do
+  if ! kubectl rollout status deployment "$deploy" --timeout 5m; then
+    echo "$deploy deployment rollout status check failed"
+    exit 1
+  fi
+done
+
+# Step 2
+# $snippet route_all_v1.sh syntax="bash" outputis="text" outputsnippet="true"
```

Comment:
There shouldn't be any `# $snippet <name>` lines in the program. That will instruct the framework to generate snippets, which we don't use anymore. It's a deprecated function of the framework.
```suggestion
# $snippet
```